components/scripts/gradle/05-validate-remote-build-caching-ci-local.sh

```diff
@@ -194,19 +194,18 @@ validate_build_config() {
 }
 
 execute_build() {
-  local args
-  args=(--build-cache --init-script "${INIT_SCRIPTS_DIR}/configure-remote-build-caching.gradle")
-  if [ -n "${remote_build_cache_url}" ]; then
-    args+=("-Ddevelocity.build-validation.remoteBuildCacheUrl=${remote_build_cache_url}")
-  fi
```

Comment:
Was this replaced by an env var somewhere? If yes, I don't immediately see where.

Reply:
Yes. I had to move it to `gradle.sh` because I didn't see a way to make `invoke_gradle` accept both extra envs and extra args.

Reply:
https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/71fe28813cad8756e73f88365190947d17baf3b9/components/scripts/lib/gradle.sh#L34-L38

Reply:
Only experiment 5 exposes an option to configure the remote build cache URL, but I can't think of a reason it would cause problems moving it to a common function.

Reply:
Ok, that makes sense.  👍